### ISSUE TITLE
A11y Fail: Data Grid - check box labels

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1384]
CENG_1958

https://react-magma.cengage.com/version/4.5.0/api/datagrid/#selectable

### Issue:
Labels used for checkboxes inside of tables that are used to select rows do not have meaningful and unique labels (i.e. "Select row undefined of undefined").

### Suggestion:
Ensure that every label is unique (i.e. Select row 2 of 10).

**Comment by silvalaura** [comment]
This has been broken since v2 😓
